add Just Juice plugin

bass3l opened the pull request:
<!-- Make sure to include your plugin name below! -->

# Just Juice

Shows the battery icon/percentage in a small manner on the left side of the display screen.

Yes, MangoPeel exists and it's super cool, but I couldn't for the life of me figure out how to only display the battery via the plugin ui :laughing: also you have to be a power user on how MangoHud works to achieve what Just Juice does in a simple click.

![screenshot](https://raw.githubusercontent.com/bass3l/decky-just-juice/master/assets/screenshot-1.png)

## Checklist:

### Developer Checklist

- [X] I am the original author or an authorized maintainer of this plugin.
- [X] I have abided by the licenses of the libraries I am utilizing, including attaching license notices where appropriate.

### Plugin Checklist

- [X] I have verified that my plugin works properly on the Stable and Beta update channels of SteamOS.
- [X] I have verified my plugin is unique or alternatively provides more/alternative functionality to a similar plugin already on the store. `(Kinda? :D)`

<!-- The following section needs to be modified as yes/no answers by the plugin developer. -->

<!-- Ex: "**Yes/No**: ..." becomes "**Yes**: ..." -->

### Plugin Backend Checklist

- **No**: I am using a custom backend other than Python.
- **No**: I am using a tool or software from a 3rd party FOSS project that does not have it's dependencies [statically linked](https://en.wikipedia.org/wiki/Static_library).
- **No**: I am using a custom binary that has all of it's dependencies statically linked.

<!-- The following section is should be modified to fit the conditions for plugin testing found here: https://wiki.deckbrew.xyz/en/plugin-dev/review-and-testing -->

## Testing

<!-- Remove this box for SteamOS Stable/Beta testing if you use a custom or remote binary, for more info follow the URL in the comment above the testing section. -->
- [X] Tested on SteamOS Stable/Beta Update Channel.

RodoMa92 commented:
Simple and easy plugin. One note tho: Please try to get the home directory using this parameter instead of hardcoding to `\home\deck`, see https://github.com/SteamDeckHomebrew/decky-plugin-template/blob/cbd489150fc2f721fe788ef6d58b8b792843e7d0/decky_plugin.pyi#L23C5-L23C6 This is fine for Deck users, but it will prevent this plugin to work on other distributions.

Good enough for testing in my book :)

bass3l commented:
@RodoMa92 Thanks, I've updated the path to use the `DECKY_USER_HOME`. But I'm not sure why the latest update triggered all these github actions :laughing: :thinking:

RodoMa92 reviewed (approved):
Looks fine to me for testing :)

suchmememanyskill reviewed (dismissed):
Testing: Cannot get the HUD to show

Steam deck oled
Steamos preview (steam 1719967667)
Decky pre-release v3.0.0-pre2

bass3l commented:
@suchmememanyskill Thank you for your test, I have update the PR with a fix.

[MangoHud v0.7.2](https://github.com/flightlessmango/MangoHud/releases/tag/v0.7.2) changed something that I was relying on (apparently it was a bug, and it's now fixed, which allowed the plugin to overwrite some config options from the preset options).

I fixed this by ensuring that we disable the `no_display` option after every change to the overlay level.

TrainDoctor reviewed (commented):
Good for testing store again after reviewing latest submodule submission.

TrainDoctor commented:
@bass3l just as a heads up, plugins without testing reports submitted have until the end of this month to collect testing reports so the plugin can be approved to go to the production store or it will be removed from testing and the PR closed.
A new PR or re-opening the PR can be submitted at any time at your convenience and the plugin is not blacklisted if you can ensure you have a tester to evaluate the plugin within 2 weeks of re-opening the PR. Thanks for your patience and understanding.

TrainDoctor commented:
> @bass3l just as a heads up, plugins without testing reports submitted have until the end of this month to collect testing reports so the plugin can be approved to go to the production store or it will be removed from testing and the PR closed. A new PR or re-opening the PR can be submitted at any time at your convenience and the plugin is not blacklisted if you can ensure you have a tester to evaluate the plugin within 2 weeks of re-opening the PR. Thanks for your patience and understanding.

Plugin PR is being closed due to lack of testing. Apologies. If you wish to resubmit your plugin with a tester ready to go later down the line please feel free to do so. Thanks for submitting your plugin regardless of this outcome.
